roadmap/implementers-guide/src/runtime/disputes.md

```diff
@@ -17,90 +19,351 @@ We account for these requirements by having the validity module handle two kinds
+## Disputing an Unavailable Candidate
 
-- A candidate
-- The session that the candidate has appeared in.
-- Backing for that candidate
-- The validation code necessary for validation of the candidate.
-  > TODO: optimize by excluding in case where code appears in `Paras::CurrentCode` of this fork of relay-chain
-- Secondary checks already done on that candidate, containing one or more disputes by validators. None of the disputes are required to have appeared on other chains.
-  > TODO: validator-dispute could be instead replaced by a fisherman w/ bond
+In the case of malicious validator sending a dispute gossip message, with a block hash, that never became part of any parachain.
+
+If the candidate was not made available on another fork of the relay chain (as in it does not exist), the availability process will time out and the disputing validator will be slashed on this fork.
+As with other disputes, the escalation
+
+> TODO: is the escalation equivalent to the `DisputeGossipMessage` or is it something separate? Which information must be contained in that.
+
+## Resolving state of availability
+
+The availbility or unavailability of a block disputed in an escalation / `DisputeGossipMessage`.
+
+```mermaid
+graph LR;
+    A[Availability]-->X[Received DisputeGossipMessage]
+    X-->D
+    A-->S[Do VRF based secondary checks]
+    S-->D
+    D-->SP[2/3 voted for invalidity]
+    D-->SC[2/3 voted against invalidity]
+    SP-->SM[Slash the opposing minority]
+    SM-->Copy[Copy the transaction to forks]
+    SC-->SM
+    D-->NM[Failed to gather majority]
+```
+
+```mermaid
+graph LR;
+    UV[Unavailability Vote Concluded]-->VT[Timed out w/o Resolution]
+    UV-->SP[2/3 pro Unavailablity]
+    UV-->SC[2/3 con Unavailablity]
+    SP-->SM[Slash Minority]
+    SC-->SM
+    VT-->SA[Slash All Validators]
+```
+
+1. Query all validators iff the disputed block is known to them.
+2. Start a timeout of `UNAVAILABILITY_TIMEOUT`
+  2. For each incoming vote:
+    2. Count the vote as either pro or con unavailability
+    2. Iff super majority is reached either way
+      2. Shorten the timeout period (in blocks)
+3. Timeout reached
+  3. Slash all remaining votes appropriately
+  3. Transplant slashes transactions to all other relay chain heads off-chain
+
+
+If the availability process passes, the _remote_ dispute is ready to be voted for.
+
+### Conclusion
+
+As with the local dispute, validators self-select based on a [VRF](##-Secondary-Approval-Checking).
+
+After enough validator self-select, under the same escalation rules as for local disputes, the _remote_ dispute will conclude, slashing all those on the wrong side of the dispute.
+
+After concluding, the remote dispute remains open for a set amount of blocks to accept any further proof of additional validators being on the wrong side.
+
+`UNAVAILABILITY_TIMEOUT` should be set generously, the validators are bound for 28 days, and this is maximum time until we slash. Recommended timeout is `1d`.
```

Comment:
Actually yes doing a timeout is waay simpler than the junk I just mentioned.  👍

Reply:
Re-reading this again, it's a bit ambiguous on what is timing out. Going to rephrase this a bit to be more precise.